scripts/check-imports.js

```diff
@@ -87,6 +88,30 @@ function assertImportResolvesWithinPkgRoot(source, pkgRoot, srcFile, node) {
   }
 }
 
+function assertNoLocalDirectoryImports(source, pkgRoot, srcFile, node) {
```

Comment:
Thanks for adding this, btw :star:

Reply:
My pleasure.  Thanks for blessing it and merging it : )